ocean-client/src/utils/program-state-converter.ts

```diff
@@ -0,0 +1,25 @@
+import { ProgramState } from "../programs/common-program";
+
+export class ProgramStateConverter {
+    static toValue(information: ProgramStateInformation): string {
+        return "" + information.state + "|" + information.tx + "|" + information.blockHeight
```

Comment:
why not  JSON.stringify ? bit more data, but better readable and adaptable in the future

Reply:
had issues with converting and parsing from initial value, then thought it might be problematic with our template setup and didn't want to bother with it, tbh.